code/game/machinery/autolathe/artist_bench.dm

```diff
@@ -0,0 +1,405 @@
+				R.penetration_multiplier = 1.2 + rand(-5,5)/10
+				R.recoil_buildup = 35 + rand(-5,5)
+
+			if("shotgun") //From bull.dm, Arbitrary values
+				R.caliber = CAL_SHOTGUN
+				R.damage_multiplier = 0.8 + rand(-2,2)/10
+				R.penetration_multiplier = 0.75 + rand(-3,3)/10
+				R.recoil_buildup = 1.2 + rand(-2,2)/10//from sawnoff.dm
+				R.one_hand_penalty = 12 + rand(-2,3)
+				R.bulletinsert_sound = 'sound/weapons/guns/interact/shotgun_insert.ogg'
+				R.fire_sound = 'sound/weapons/guns/fire/shotgunp_fire.ogg'
+
+			if("rifle")
+				R.caliber = pick(CAL_CLRIFLE, CAL_SRIFLE, CAL_LRIFLE)
+				R.fire_sound = 'sound/weapons/guns/fire/smg_fire.ogg'
+
+			//No gun currently uses CAL_357 far as I know
+			//	if("revolver")
+			//		caliber = pick(CAL_357)
+
+			if("sniper")//From sniper.dm, Arbitrary values
+				R.caliber = CAL_ANTIM
+				R.bulletinsert_sound = 'sound/weapons/guns/interact/rifle_load.ogg'
+				R.fire_sound = 'sound/weapons/guns/fire/sniper_fire.ogg'
+				R.one_hand_penalty = 15 + rand(-3,5) //From sniper.dm, Temporary values
+				R.recoil_buildup = 90 + rand(-10,10)
+
+			if("gyro")//From gyropistol.dm, Arbitrary values
+				R.caliber = CAL_70
+				R.recoil_buildup = 0.1 * rand(1,20)
+
+			if("cap")
+				R.caliber = CAL_CAP
+
+			if("rocket")//From RPG.dm, Arbitrary values
+				R.caliber = CAL_ROCKET
+				R.fire_sound = 'sound/effects/bang.ogg'
+				R.bulletinsert_sound = 'sound/weapons/guns/interact/batrifle_magin.ogg'
+				R.one_hand_penalty = 15 + rand(-3,5)//From ak47.dm, temporary values
+				R.recoil_buildup = 15 + rand(-3,3)
+
+			if("grenade")
+				R.caliber = CAL_GRENADE
+				R.fire_sound = 'sound/weapons/guns/fire/grenadelauncher_fire.ogg'
+				R.bulletinsert_sound = 'sound/weapons/guns/interact/batrifle_magin.ogg'
+				R.one_hand_penalty = 15 + rand(-2,3)//from sniper.dm, Temporary values
+				R.recoil_buildup = 20 + rand(-5,5) //from projectile_grenade_launcher.dm
+
+		if(R.max_shells == 3 && (gun_pattern == "shotgun"||"rocket"))//From Timesplitters triple-firing RPG far as I know
+			R.init_firemodes = list(
+				list(mode_name="fire one barrel at a time", burst=1, icon="semi"),
+				list(mode_name="fire three barrels at once", burst=3, icon="auto"),
+				)
+		return R
+
+	else if(full_artwork == "artwork_statue")
+		var/obj/structure/artwork_statue/S = new(src)
+		return S
+
+	else if(full_artwork == "artwork_oddity")
+		var/obj/item/weapon/oddity/artwork/O = new(src)
+		var/list/oddity_stats = list(STAT_MEC = rand(0,1), STAT_COG = rand(0,1), STAT_BIO = rand(0,1), STAT_ROB = rand(0,1), STAT_TGH = rand(0,1), STAT_VIG = rand(0,1))//May not be nessecary
+		var/stats_amt = 3
+		if(ins_used >= 85)//Arbitrary values
+			stats_amt += 3
+		if(ins_used >= 70)
+			stats_amt += 3
+		if(ins_used >= 55)
+			stats_amt += 3//max = 3*4*2+6 = 30 points, min 3*4+6 = 18
+		for(var/i in 1 to stats_amt)
+			var/stat = pick(ALL_STATS)
+			oddity_stats[stat] = min(MAX_STAT_VALUE, oddity_stats[stat]+rand(1,2))
```

Comment:
Okay so we add 1-2 to a random stat 3-12 times. And we can't go over the MAX_STAT_VALUE on the particular oddity. alright.